workflows/cmgg-wisecondorx.nf

```diff
@@ -48,7 +46,11 @@ include { INPUT_CHECK } from '../subworkflows/local/input_check'
 //
 // MODULE: Installed directly from nf-core/modules
 //
-include { FASTQC                      } from '../modules/nf-core/fastqc/main'
+
+include { SAMTOOLS_FAIDX              } from '../modules/nf-core/samtools/faidx/main'
+include { SAMTOOLS_INDEX              } from '../modules/nf-core/samtools/index/main'
+include { WISECONDORX_CONVERT         } from '../modules/nf-core/wisecondorx/convert/main'
+include { WISECONDORX_NEWREF          } from '../modules/nf-core/wisecondorx/newref/main'
 include { MULTIQC                     } from '../modules/nf-core/multiqc/main'
```

Comment:
also drop MQC?

Reply:
It's only used to show a summary of the versions used. I can remove it if that's not necessary